src/shared/components/navbar/desktop/index.js

```diff
@@ -7,34 +7,47 @@ import PropTypes from 'prop-types'
 import Link from 'shared/components/link'
 import styles from './index.module.css'
 
-const DesktopNavbar = ({ isSticky, localesBarHeight, intl: { messages } }) => {
-  const navbarClasses = classNames(styles.container,
-    {
-      [styles.sticky]: isSticky
-    })
+class DesktopNavbar extends Component {
+  render () {
+    const { isSticky, intl: { messages } } = this.props
+    const navbarClasses = classNames(styles.container,
+      {
+        [styles.sticky]: isSticky
+      })
 
-  return (
-    <div className={ navbarClasses } >
-      <div className={ styles.navbarMenu }>
-        <Helmet>
-          <script async defer src="https://buttons.github.io/buttons.js" />
-        </Helmet>
-        <Link to="/"> { messages.navbar.item1 } </Link>
-        <Link to="/"> { messages.navbar.item2 } </Link>
-        <Link to="/"> { messages.navbar.item3 } </Link>
-        <Link to="/"> { messages.navbar.item4 } </Link>
-        <a className="github-button" href="https://github.com/ipfs/js-ipfs" data-show-count="true" aria-label="Star ipfs/js-ipfs on GitHub">
-            Star
-        </a>
+    return (
+      <div className={ navbarClasses } >
+        <div className={ styles.navbarMenu }>
+          <Helmet>
+            <script async defer src="https://buttons.github.io/buttons.js" />
+          </Helmet>
+          <div className={ styles.link } onClick={ this.handleGettingStartedClick }> { messages.navbar.item1 } </div>
```

Comment:
Couldn't we use a `<a>` pointing to a anchor? We could then call `event.preventDefault()` to stop it going down instantly and to the animation regularly. The advantage is that the link is copy pastable.